src/main/kotlin/org/rust/cargo/project/configurable/CargoConfigurable.kt

```diff
@@ -17,18 +17,23 @@ import com.intellij.ui.dsl.builder.panel
 import com.intellij.ui.dsl.gridLayout.HorizontalAlign
 import org.rust.RsBundle
 import org.rust.cargo.project.model.isNewProjectModelImportEnabled
+import org.rust.cargo.project.settings.RustProjectSettingsService
+import org.rust.cargo.project.settings.rustSettings
 import java.awt.Component
 
 class CargoConfigurable(
     project: Project,
     private val isPlaceholder: Boolean
 ) : RsConfigurableBase(project, RsBundle.message("settings.rust.cargo.name")) {
+    private val settings: RustProjectSettingsService = project.rustSettings
```

Comment:
It seems you can move it into `createSettingsPanel` or even call each time - service `getInstance` is quite cheap